src/lib.rs

```diff
@@ -123,7 +123,7 @@ pub use serde_json::value::Value;
 ///
 /// A PyException can be returned as an error, it is converted to a Python `Exception`, and the
 /// message will be used as the exception's arguments.
-/// If an error is thrown, it is converted to a Python `RuntimeError`, and the `Debug` string for
+/// If an Error type is returned, it is converted to a Python `RuntimeError`, and the `Display` string for
```

Comment:
I changed the wording here for better understanding and clarity. Throwing errors has a very different meaning depending on what programming language you may have learned before :)